psse/psse-converter/src/main/java/com/powsybl/psse/converter/TransformerConverter.java

```diff
@@ -247,6 +249,14 @@ private void createThreeWindingsTransformer() {
         defineOperationalLimits(twt, voltageLevel1.getNominalV(), voltageLevel2.getNominalV(), voltageLevel3.getNominalV());
     }
 
+    private String getNameOrNull() {
+        String name = psseTransformer.getName();
+        if (name.trim().isEmpty()) {
```

Comment:
There's no check in `PsseTransformer.setName(String)` to ensure that the name is not `null`.
I think it would be safer to consider that the name can be `null` here:

```suggestion
        if (name != null && name.trim().isEmpty()) {
```